apps/web/src/scenes/UserGalleryPage/UserSharedInfo/UserSharedInfoList/SharedInfoListRow.tsx

```diff
@@ -33,11 +35,17 @@ export default function SharedInfoListRow({ title, subTitle, href, imageContent
     );
   }, [imageContent, subTitle, title]);
 
+  const onPressUserRow = useCallback(() => hideModal(), [hideModal]);
```

Comment:
another extreme nit: when it comes to react naming conventions, these callbacks should be defined as `handleSomeAction` whereas prop names are defined with `onSomeAction`

therefore should look like: `<StyledRowNonLink onClick={handlePressUserRow} ...`

Reply:
<img width="655" alt="image" src="https://github.com/gallery-so/gallery/assets/12162433/c58e5bd5-6391-40c7-8358-62a28f43bd3c">
https://jaketrent.com/post/naming-event-handlers-react/